libmcount/wrap.c

```diff
@@ -615,6 +617,17 @@ __visible_default int fexecve(int fd, char *const argv[], char *const envp[])
 	return real_fexecve(fd, argv, new_envp);
 }
 
+__visible_default int close(int fd)
+{
+	if (unlikely(real_close == NULL))
+		mcount_hook_functions();
+
+	if (unlikely(fd == pfd)) {
+		return 0;
+	}
+
+	return real_close(fd);
+}
```

Comment:
Nitpick: please add a blank line between the closing bracket and the `#ifdef`.

Reply:
updated the code & commit message